scripts/migrateImageDeliveryChannels.sql

```diff
@@ -0,0 +1,51 @@
+-- convert image defaults
+INSERT INTO "ImageDeliveryChannels" ("ImageId", "Channel", "DeliveryChannelPolicyId")
+SELECT images."Id", 'iiif-img', (SELECT "DeliveryChannelPolicies"."Id" from "DeliveryChannelPolicies" WHERE ("DeliveryChannelPolicies"."Customer", "Channel", "DeliveryChannelPolicies"."Name") = (1, 'iiif-img', 'default'))
+    FROM (SELECT * FROM  "Images" WHERE  "Images"."DeliveryChannels" LIKE '%iiif-img%' AND "Images"."ImageOptimisationPolicy" <> 'use-original' AND "NotForDelivery" = false) AS images

```

Comment:
There's an extra/unnecessary nesting here. LHS of union can be

```sql
SELECT "Id", 'iiif-img', (SELECT....)
FROM "Images"
WHERE "DeliveryChannels" LIKE '%iiif-img%' 
 AND "ImageOptimisationPolicy" <> 'use-original' 
 AND "NotForDelivery" = false
```

Think this is a case for a couple of the queries here.